services/web-app/pages/api/log-message.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { Logging } from '@carbon-platform/api/logging'
+
+const logging = new Logging('web-app', 'log-message-api')
```

Comment:
Let's make this API very specific to logging incoming requests. Maybe name it `log-request.js` and change this component in the constructor here to `log-request-api`.

Reply:
Done!